### DISCUSSION TITLE
Does Port 8555 need to open to receive reward?

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#3686] [Q&A]
Cannot get port 8555 to open. Tried the router setting and played with windows firewall setting. Not working. Would this affect the chance of getting reward?

Any guide to set this up properly under windows?

**Comment by justinphelps** [comment]
8555 is the full_node RPC port and shouldn't be available to the internet. It's not the port that you're looking for and really should  only be available to localhost. 
8444 is the port that needs to be available for the chia network to communicate to your full node . Either by configuring your router for port forwarding or uPNP.

**Reply by pluto2021** [reply]
Got it. Thanks for the help.  How to check if the port 8555 is functioning properly? or if it is not configured properly, what kind of errors or warnings you would see?

**Comment by fisch88** [comment]
I also have a question on this topic: if I have 2 machines on the same network with chia and 2 different wallets, both connected and in the farm without errors, does it mean that the connection was successful or are there other things to check? Obviously, only on one of the 2 nodes will I be able to hijack the ports ..
